src/rest/routes/github-callback/service.ts

```diff
@@ -0,0 +1,34 @@
+import { Request } from "express";
+import { envVars } from "config/env";
+import axios from "axios";
+
+const GITHUB_CLOUD_LOGIN_URL = "https://github.com/login/oauth/access_token";
+export const getAccessToken = async (req: Request) => {
```

Comment:
This is a pure business service file, so let's try not to put any http-related classes here, like `Request`. Just pass in the param as normal args.

Reply:
Updated